src/main/java/com/marginallyclever/makelangelo/Makelangelo.java

```diff
@@ -284,11 +285,10 @@ private void estimateTime() {
 	}
 
 	private void openPlotterControls() {
-		PlotterControls plotterControls = new PlotterControls(myPlotter,myTurtle);
-		JDialog dialog = new JDialog(mainFrame,PlotterControls.class.getSimpleName());
+		JDialog dialog = new JDialog(mainFrame, Translator.get("PlotterControls.Title"));
+		PlotterControls plotterControls = new PlotterControls(myPlotter,myTurtle, dialog);
```

Comment:
why does PlotterControls need dialog?  for a parent component?

Reply:
yes, it is need as a parent